### DISCUSSION TITLE
Overriding shapes from modules

### DISCUSSION THREAD

**Discussion by craftyweb** [OrchardCMS/OrchardCore#13988] [Q&A]
Hi,

I'm trying to override the ContentsAdminList shape to add an extra zone with custom filters for certain ContentTypes. I've added the shape to my custom module and I've made a reference to OrchardCore.Contents (original module of the shape).

When I run the website the default shape is still shown. When I go a little bit deeper into the code I can see that the file is found
![image](https://github.com/OrchardCMS/OrchardCore/assets/61787638/0bd0b238-a901-439b-b638-2faf051e03cc)

I have no idea why it's not working. Does anyone have a solution for this?

Thanks!

**Comment by Psichorex** [comment]
@craftyweb Hi, I just came across this problem. It did not work from 2 different modules. I tried a 3rd one and it worked. No clue why this is exactly happening.

**Comment by ns8482e** [comment]
Most likely the issue with the order of module being loaded. Add original module as dependency in your module. Also make sure that your module is enabled.
